site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -215,17 +215,19 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.allocator.updateStrategy`                        | The [strategy](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) to apply to the ping deployment                                                                                          | `{}`                               |
 | `agones.ping.updateStrategy`                             | The [strategy](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) to apply to the allocator deployment                                                                                     | `{}`                               |
 
-{{% feature publishVersion="1.28.0" %}}
-**New Configuration Features:**
-
-| Parameter                             | Description                                                                                                                                                                                                                           | Default |
-|---------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| `agones.allocator.pdb.enabled`        | Set to `true` to enable the creation of a [PodDisruptionBudget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) for the allocator deployment                                                                         | `false` |
```

Comment:
I think the line above needs to be removed. See: https://github.com/googleforgames/agones/blob/59eed9a6aed9f69ffe4ee0d3b9ccb6fc89caf1e7/site/content/en/docs/Installation/Install%20Agones/helm.md

The table isn't rendering correctly.

Also, can you group the new config option in the same area as other please? So put the new `agones.allocator` config options with the others, the new `agones.ping` options with the others, etc.